src/extension/extension.ts

```diff
@@ -24,186 +44,205 @@ export function activate(context: vscode.ExtensionContext): void {
     }
     const solc: string =
       vscode.workspace.getConfiguration().get('SolcExecutable') || ''
-
-    const solcArgs: string =
-      vscode.workspace.getConfiguration().get('SolidityArguments') || ''
+    const solcArgs: string = JSON.stringify(
```

Comment:
why using stringify if I use parse later?

Reply:
because I have to change format of object via JSON.parse(JSON.stringfy()) or it won't recognize it